packages/carousel/README.md

```diff
@@ -0,0 +1 @@
+# pcln-carousel
```

Comment:
nb: documentation for how to use the component

Reply:
That'll be included in the next PR to add the rest of the props we discussed.